low-level/framework/sspl_init

```diff
@@ -21,10 +21,8 @@ SCRIPT_NAME=$(basename $0)
 source $script_dir/constants.sh
```

Comment:
we need to remove this file all together, legacy code staying here and confusing. anyways changes here am not bothered much